docs/list.md

```diff
@@ -197,6 +197,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Sign in with [Github here](https://devict-hacktoberfest.herokuapp.com/) so they can track your PRs.
 - **Notes**: You need to create at least two PRs in any of the  Wichita developer community organizations listed [here](https://devict-hacktoberfest.herokuapp.com/)
 
+#### **devtron**
+
+- **Swag**: Devtron's Limited Edition Swag Kit for 1st Rank , Devtron's Limited Edition Diary + stickers for 2nd Rank,  Devtron's Limited Edition stickers for 3rd Rank
+- **Requirements**:
+    - Star the [Devtron](https://github.com/devtron-labs/devtron) repo on Github
+    - Follow us on [Twitter](https://twitter.com/DevtronL)
+    - Make some art and share it on Twitter. Tag @DevtronL and use    #Hacktoberfest, #Devtron on your post.
+- **How to sign up**: Open this [Github Repo](https://github.com/devtron-labs/devtron) ,Fork it and star it . When submitting a PR please add Hacktoberfest in the PR Description. This would give us a confirmation on whether the PR is for Hacktoberfest.
+- **Issues**: Checkout [Devtron Repo](https://github.com/devtron-labs/hacktoberfest2022/blob/main/README.md#book-contribution-guidelines).
```

Comment:
The Issues section, if included (as it's optional), is when an organization wants to highlight certain GitHub Issues as counting for Hacktoberfest. If that is not the case, then it should not be included here.